calm_adapter/calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/CalmStore.scala

```diff
@@ -24,7 +24,10 @@ class CalmStore(store: VersionedStore[String, Int, Map[String, String]]) {
   def shouldStoreRecord(record: CalmRecord): Result[Boolean] =
     store
       .getLatest(record.id)
-      .map { case Identified(_, storedData) => record.data != storedData }
+      .map {
+        case Identified(_, storedRecord) =>
+          record.retrievedAt.toEpochMilli > storedRecord.retrievedAt.toEpochMilli
```

Comment:
Maybe use the comparison helper on `java.time.Instant`?

```suggestion
          record.retrievedAt.isAfter(storedRecord.retrievedAt)
```